packages/tf-inventory/src/stateFile.js

```diff
@@ -178,7 +180,12 @@ async function listResourcesForFile(file) {
     let ec2Instances = await Promise.all(ec2InstancePromises);
     ec2Instances = [].concat(...ec2Instances);
 
-    return { ecsClusters, ec2Instances };
+    let esDomainNames = stateFile.resources
+      .filter((r) => r.type === 'aws_elasticsearch_domain')
+      .map((c) => c.instances.map((i) => i.attributes.domain_name));
```

Comment:
So this returns a multi-dimensional array, correct?

Reply:
yes